projects/valtimo/plugin/src/lib/plugins/smart-documents/components/generate-document-configuration/generate-document-configuration.component.html

```diff
@@ -40,6 +41,7 @@
     [defaultValue]="obs.prefill?.templateName"
     [disabled]="obs.disabled"
     [required]="true"
+    placeholder=""
```

Comment:
What's the point of these empty strings for placeholder? I hope the input is not required?